common/lib/xmodule/xmodule/modulestore/mongo/base.py

```diff
@@ -1958,16 +1958,24 @@ def ensure_indexes(self):
                 ('_id.name', pymongo.ASCENDING),
                 ('_id.revision', pymongo.ASCENDING),
             ],
-            background=True)
+            # While this index would more safely be "background=True", it's not currently created that way.
```

Comment:
What if we wrapped this in a try/except block for each index creation and set background=True? That way if the index exists it won't be touched which is good from an operational view, all the indexes would be attempted to be created, and if they do not currently exist they will be created in the background.